RecoEgamma/PhotonIdentification/python/Identification/mvaPhotonID_RunIIFall17_v1_cff.py

```diff
@@ -0,0 +1,125 @@
+MVA_WP80 = PhoMVA_2Categories_WP(
+    idName = idName,
+    mvaValueMapName = mvaValueMapName,           # map with MVA values for all particles
+    mvaCategoriesMapName = mvaCategoriesMapName, # map with category index for all particles
+    cutCategory0 = 0.67,  # EB new val : sig eff = 80% , bkg eff = ?
+    cutCategory1 = 0.54   # EE new val : sig eff = 80% , bkg eff = ?
+    )
+
+#
+# Finally, set up VID configuration for all cuts
+#
+
+# Create the PSet that will be fed to the MVA value map producer
+mvaPhoID_RunIIFall17_v1_producer_config = cms.PSet( 
+    mvaName            = cms.string(mvaFall17v1ClassName),
+    mvaTag             = cms.string(mvaTag),
+    weightFileNames    = mvaRunIIFall17WeightFiles_V1,
+    #
+    # All the event content needed for this MVA implementation follows
+    #
+    # All the value maps: these are expected to be produced by the
+    # PhotonIDValueMapProducer running upstream
+    #
+    phoChargedIsolation = cms.InputTag("photonIDValueMapProducer:phoChargedIsolation"),
+    phoPhotonIsolation  = cms.InputTag("photonIDValueMapProducer:phoPhotonIsolation"),
+    phoWorstChargedIsolation = cms.InputTag("photonIDValueMapProducer:phoWorstChargedIsolation"),
+    #
+    # Original event content: pileup in this case
+    # 
+    rho                       = cms.InputTag("fixedGridRhoAll"), # As used by Hgg and by developer of this ID
+    # In this MVA for endcap the corrected photon isolation is defined as
+    # iso = max( photon_isolation_raw - rho*effArea - coeff*pt, cutoff)
+    # as discussed in the indico presentations listed in the beginning of this file.
+    #
+    effAreasConfigFile = cms.FileInPath(effAreasPath_pho),
+    # The coefficients "coeff" for the formula above for linear pt scaling correction
+    # the first value is for EB, the second is for EE
+    # NOTE: even though the EB coefficient is provided, it is not presently used in the MVA.
+    # For EB, the uncorrected raw photon isolation is used instead.
+    phoIsoPtScalingCoeff = cms.vdouble(0.0035,0.0040)
+    # The cutoff for the formula above
+    # phoIsoCutoff = cms.double(2.5)
+    )
+
+# Create the VPset's for VID cuts
+mvaPhoID_RunIIFall17_v1_wp90 = configureVIDMVAPhoID_V1( MVA_WP90 )
+mvaPhoID_RunIIFall17_v1_wp80 = configureVIDMVAPhoID_V1( MVA_WP80 )
+
+# The MD5 sum numbers below reflect the exact set of cut variables
+# and values above. If anything changes, one has to 
+# 1) comment out the lines below about the registry, 
+# 2) run "calculateMD5 <this file name> <one of the VID config names just above>
+# 3) update the MD5 sum strings below and uncomment the lines again.
+#
+
+central_id_registry.register( mvaPhoID_RunIIFall17_v1_wp90.idName,
+                              '0e737fa7da64b5e6e058130f783a5a64')
+central_id_registry.register( mvaPhoID_RunIIFall17_v1_wp80.idName,
+                              '67f3e11869b69ad650e993a443019e85')
+
+mvaPhoID_RunIIFall17_v1_wp90.isPOGApproved = cms.untracked.bool(False)
```

Comment:
similar to a few other places: I think that unapproved IDs do not belong in an official release.
OK for recipes.

Reply:
Changes implemented here:
https://github.com/lsoffi/cmssw/blob/CMSSW_9_4_0_pre3_TnP/RecoEgamma/PhotonIdentification/python/Identification/mvaPhotonID_RunIIFall17_v1_cff.py
These V1 IDs are approved and ready to be merged in CMSSW.